insightfinder/assets/dashboards/ifdashboard.json

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
The logo image is not loading for me when i open the dashboard, so I wanted to confirm if it's working on your end?You may want to remove and re-add the image.

Reply:
Unfortunately this image is still not loading for me - maybe another image would work?

Reply:
@anawishnoff - I see it for first time and then it never loads. I tried few more images. I am using a dropbox link. Do you recommend any other place to load the image and use that link?

Reply:
@ajainIF It may be an issue loading it from Dropbox. I see many other integrations host the image somewhere with their website, is that an option for your team?

Reply:
That was a great suggestion. Can you check now?